test/string_test.rb

```diff
@@ -273,6 +273,29 @@
     end
   end
 
+  describe '#singularize' do
+    before do
+      @singular, @plural = *TEST_SINGULARS.to_a.sample
```

Comment:
@joneslee85 I agree with you, it is strange. But, considering that we've a bunch of fixtures, it doesn't makes more sense to use them?